plugins/Live/Live.php

```diff
@@ -19,6 +19,9 @@
  */
 class Live extends \Piwik\Plugin
 {
+    protected static $visitorProfileEnabled = null;
+    protected static $visitorLogEnabled = null;
+    protected static $siteIdLoaded = null;
```

Comment:
Note: if adding static variables like this, we'd need to make sure they are unset in `Fixture::clearInMemoryCaches()` (though these won't affect any existing tests I think, but they might in the future). If using the transient cache, then nothing needs to change.

Reply:
using transient cache instead sound fine... will change that

Reply:
done